pkg/routing/redisrouter.go

```diff
@@ -207,6 +207,7 @@ func (r *RedisRouter) WriteNodeRTC(_ context.Context, rtcNodeID string, msg *liv
 }
 
 func (r *RedisRouter) startParticipantRTC(ss *livekit.StartSession, participantKey livekit.ParticipantKey, participantKeyB62 livekit.ParticipantKey) error {
+	prometheus.IncrementParticipantJoin(1, "rtc_init")
```

Comment:
Added these to the specific router rather than roommanager.StartSession callback since it seems there's non-trivial logic in redisrouter that we want to ensure completes successfully before the roommanger callback happens.